Perlang.ConsoleApp/Program.cs

```diff
@@ -334,14 +334,6 @@ private void Error(Token token, string message)
 
         private void ParseError(ParseError parseError)
         {
-            if (parseError.ParseErrorType == ParseErrorType.MISSING_TRAILING_SEMICOLON)
-            {
-                // These errors are ignored; we will get them all them when we try to parse expressions as
-                // statements.
-                hadError = true;
-                return;
-            }
-
```

Comment:
:point_up: This is the essence of the bug fix. It would allow _other_ cases of missing trailing semicolons (like `hej hej`) to slip through. The special-casing doesn't seem to be needed any more anyway, so we'd better get rid of it.